### PR TITLE
Minor typo

### PR DESCRIPTION
**Pull request by rhyspowell**

### CODE REVIEW

File: NFT_Collection/Section_3/Lesson_1_What_Is_On_Chain_Data.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ But, assuming the blockchain stays up forever — our NFT will be up forever! Th
  🖼  What are SVGs
 -------------------
 
-A common way to store NFT data for images is using an SVG. And SVG is an image, but the image itself is built with code.
+A common way to store NFT data for images is using an SVG. A SVG is an image, but the image itself is built with code.
```
Comment (adilanchian) [inline NFT_Collection/Section_3/Lesson_1_What_Is_On_Chain_Data.md:15]: Can you update `images is using an SVG` to `images is using a SVG` ?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Thank you!! Please see minor tweak :)

File: NFT_Collection/Section_3/Lesson_1_What_Is_On_Chain_Data.md (hunk 1 above)
Comment (rhyspowell) [reply NFT_Collection/Section_3/Lesson_1_What_Is_On_Chain_Data.md]: fixed, sorry completely missed that one!

**Review by adilanchian — APPROVED** [review]

**Comment by adilanchian** [comment]
LEGEND. Thanks so much for this!

**Comment by adilanchian** [comment]
@all-contributors please add @rhyspowell for doc

**Pull request merged**